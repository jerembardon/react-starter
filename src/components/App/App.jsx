import React, { Component } from 'react';

// eslint-disable-next-line react/prefer-stateless-function
class App extends Component {
  render() {
    return (
      <React.Fragment>
        <h1> React Webpack Starter </h1>
        <p> Select module you want to install </p>
      </React.Fragment>
    );
  }
}

export default App;
