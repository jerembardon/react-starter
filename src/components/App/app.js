import React, {Component} from 'react';
import pool from './../../assets/img/pool.jpg';
import './../../assets/style/app.scss'
class App extends Component {
    render() {
        return(
            <React.Fragment>
                <h1> React Webpack Starter </h1>
                <p> Select module you want to install </p>
            </React.Fragment>
        )
    }  
}

export default App;