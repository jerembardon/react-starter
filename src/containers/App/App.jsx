import React, { Component } from 'react';
import {Button} from './../../components/Button/index'

// eslint-disable-next-line react/prefer-stateless-function
class App extends Component {
  render() {
    return (
      <React.Fragment>
        <h1> React Webpack Starter </h1>
        <p> Select module you want to install </p>
        <Button> Hello </Button>
      </React.Fragment>
    );
  }
}

export default App;
