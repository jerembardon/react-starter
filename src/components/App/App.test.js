import React from 'react';
import { shallow } from 'enzyme';

import App from "./App"
import pool from "./../../assets/img/pool.jpg";

describe('App rendering test', () => {
    it('Should render app', () => {
      const wrapper = shallow(<App />);
      expect(wrapper).toMatchSnapshot();
    });
  });