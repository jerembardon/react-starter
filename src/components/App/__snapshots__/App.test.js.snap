// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendering test Should render app 1`] = `ShallowWrapper {}`;
